web/src/test/java/org/springframework/security/web/util/matcher/MediaTypeRequestMatcherTests.java

```diff
@@ -79,6 +80,16 @@ public void constructorEmtpyMediaTypes() {
 				Collections.<MediaType> emptyList());
 	}
 
+	@Test(expected = IllegalArgumentException.class)
+	public void constructorEmptyArgs() {
```

Comment:
I realize that the older tests here aren't named very well, but would you mind changing your test names so that it's clear what it being tested?

For new tests, we've been following an `xWhenYThenZ` naming convention.

Reply:
ok, it is good, i think we can update this suggestion to the `CONTRIBUTING.md`